lib/drupal.js

```diff
@@ -32,7 +32,7 @@ module.exports = function(grunt) {
   };
 
   module.majorVersion = function() {
-    var version = grunt.config('drupal.drupal-version') || '',
+    var version = process.env.GDT_DRUPAL_CORE || grunt.config('drupal.drupal-version') || '',
```

Comment:
My understanding is the current use of this variable is to route the tests to determine which set of supporting assets to use. For functional purposes we take the version from the build Drupal core codebase. Are we using the env var for something operational? I ask this more to ensure I understand than to object, though it probably bears documentation if we expect this to be used outside of the GDT self-tests.